### PR TITLE
staticcheck

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: corefoundation_1.10.go [hunk 1]
```diff
@@ -239,8 +239,8 @@ func Convert(ref C.CFTypeRef) (interface{}, error) {
 				return nil, err
 			}
 			results = append(results, v)
-			return results, nil
 		}
+		return results, nil
```
Comment (heronhaye) [inline corefoundation_1.10.go]: this is behavioral change?

Comment (heronhaye) [reply corefoundation_1.10.go]: although, it does seem like the old behavior was wrong. how did you find this? staticcheck doesnt report an error here

Comment (heronhaye) [reply corefoundation_1.10.go]: it seems reasonable to me but maybe worth to do a test darwin build just to make sure..

Comment (joshblum) [reply corefoundation_1.10.go]: static check reported this for me, i'll do some test builds with the revendored libs

Comment (heronhaye) [reply corefoundation_1.10.go]: cool, I made a ticket for vendoring everything in. But idk where it went during the squad migration lol

**Pull request merged**